### ISSUE TITLE
[Software Addition] Crystal Launcher

### ISSUE THREAD

**Issue by RDKRACZ** [NordicGamerFE/usefulmods#51]
### Checklist

- [X] I've checked that my software suggestion hasn't already been added to one of the lists.
- [X] I've checked the latest issues to make sure that this software hasn't already been suggested.

### What is your software's name?

Crystal Launcher

### What systems do you know your software works on?

- [X] Windows 11
- [X] Windows 10
- [X] Windows 8/8.1
- [X] Windows 7
- [ ] Windows XP or earlier
- [X] MacOS
- [X] Linux
- [X] Other

### Input a link to your suggested software.

https://crystal-launcher.net/

**Comment by ChiefArug** [comment]
FYI thats a pirate launcher.

**Comment by RDKRACZ** [comment]
Yes, it can be used for this but you can also use a "premium" account like in any other launcher.
There is no rule in the readme preventing suggestion of such software (yet).
The launcher has good Forge/Fabric loader installer built-in and unlike in MultiMC it will use a proper Java version for a MC version automatically.
